css/design_system/components/footer.scss

```diff
@@ -0,0 +1,100 @@
+ /**
+ * @title Simple Footer
+ * @category components
+ * @component_type footer
+ * @value s-footer
+ * @description Simple footer with paragraph text only
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ * </footer>
+ */
+
+ .s-footer {
+  background-color: var(--s-color-gray-800);
+  padding: var(--s-space-6) var(--s-space-12) var(--s-space-8);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  p {
+    @extend .s-m-0;
+    color: var(--s-color-gray-200);
+    text-align: start;
+  }
+
+  a {
+    @extend .s-text-uppercase;
+    @extend .s-text-body-sm;
+    @extend .s-font-bold;
+    text-decoration: none;
+    color: var(--s-color-gray-200);
+    letter-spacing: var(--s-space-1);
+    cursor: pointer;
+  }
+
+  a:hover {
+    color: var(--s-color-white);
+  }
+
+  .s-footer-right a {
+    @extend .s-ml-4;
+  }
+
+  .s-footer-left a {
+    @extend .s-mr-4;
+  }
+
+  img {
+    height: 22px;
+  }
+ }
+
+ /**
+ * @title Footer with Links
+ * @category components
+ * @component_type footer
+ * @value s-footer-l
+ * @description Footer with paragraph text and one set of links. Use of the <code>.s-footer-right</code> class ensures proper spacing between links.
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ *  <div class="s-footer-right">
+ *    <a href="/privacy">Privacy Policy</a>
+ *    <a href="https://turing.edu/programs/" target="blank">Programs</a>
+ *    <a href="https://turing.edu/try-coding/">Try Coding</a>
+ *  </div>
+ * </footer>
+ */
+
+.s-footer-l {
+  background-color: var(--s-color-gray-800);
```

Comment:
if I left it as an empty rule, it didn't seem to be read by the doc builder, so I just added this line in.